tree/dataframe/src/RDFInterfaceUtils.cxx

```diff
@@ -142,11 +142,17 @@ static ColumnNames_t FindUsedColumns(const std::string &expr, const ColumnNames_
    return usedCols;
 }
 
-static ParsedExpression ParseRDFExpression(const std::string &expr, const ColumnNames_t &treeBranchNames,
+static ParsedExpression ParseRDFExpression(std::string_view expr, const ColumnNames_t &treeBranchNames,
                                            const ColumnNames_t &customColNames, const ColumnNames_t &dataSourceColNames,
                                            const std::map<std::string, std::string> &aliasMap)
 {
-   const auto usedColsAndAliases = FindUsedColumns(expr, treeBranchNames, customColNames, dataSourceColNames, aliasMap);
+   // transform `#var` into `__rdf_sizeof_var`
+   TString preProcessedExpr(expr);
+   TPRegexp colSizeReplacer("(^|\\W)#([a-zA-Z_][a-zA-Z0-9_]*)"); // match #varname at beginning or after not-a-word
```

Comment:
It just occurs to me that we interfere with the pre-processor. Not sure if the pre-processor is a thing in jitted code but I could imagine it being used for OMP pragmas, for instance.

Reply:
argh, you are right. in principle you can use preprocessor directives and this PR breaks it. in practice there is a workaround that is `Declare`-ing a function (with preproc. directives if needed) beforehand and simply invoke it in the string that you pass to RDF. so...rare usecase, _but_ will be broken without notice, _but_ there is a possible workaround. not sure how to proceed :confused:

Reply:
I'd say we document it and move on.

Reply:
Perhaps one more thing: in lambdas, users would need to go back to `__rdf_sizeof_var`, right?  Users using lambdas might be ok with that, just to clarify.

Reply:
not sure I understand this last comment, you can totally write `df.Define("x", [] (std::size_t s) { ... }, {"#var"})`

Reply:
Oh. Yes.